auth/src/main/res/values-v26/styles.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:ignore="ResourceName">
+
+    <style name="FirebaseUI.TextInputEditText.EmailField">
+        <item name="android:autofillHints">emailAddress</item>
```

Comment:
Hmm I wonder how this will play with smartlock.  It could be annoying if they just turned down the smartlock hint and then autofill offers them a new hint.